bin/eslint.js

```diff
@@ -4,4 +4,8 @@ const CODE_DIR = "/code";
 process.chdir(CODE_DIR);
 
 const ESLint = require("../lib/eslint");
-ESLint.run({ dir: CODE_DIR });
+const issues = ESLint.run({ dir: CODE_DIR });
```

Comment:
> There is a major difference here from the previous behaviour, as it now accumulates all issues before printing them all. I hope it is OK.

It is very much not preferred. There can be significant performance penalties for buffering this way.

Was there no reasonable alternative?